crates/bevy_mesh/src/mesh.rs

```diff
@@ -1567,4 +1690,23 @@ mod tests {
             mesh.triangles().unwrap().collect::<Vec<Triangle3d>>()
         );
     }
+
+    #[cfg(feature = "serialize")]
+    #[test]
+    fn serialize_deserialize_mesh() {
+        let mut mesh = Mesh::new(
+            PrimitiveTopology::TriangleList,
+            RenderAssetUsages::default(),
+        );
+
+        mesh.insert_attribute(
+            Mesh::ATTRIBUTE_POSITION,
+            vec![[0., 0., 0.], [2., 0., 0.], [0., 1., 0.], [0., 0., 1.]],
+        );
+        mesh.insert_indices(Indices::U16(vec![0, 1, 2, 0, 2, 3]));
+
+        let serialized_mesh = SerializedMesh::from_mesh(mesh.clone());
```

Comment:
Could we serialize this to a string and back here as well? Just to make sure we keep this being serializable and to give an example of this.

Reply:
Good idea :)